Sources/Momento/CacheClient.swift

```diff
@@ -351,17 +353,17 @@ public class CacheClient: CacheClientProtocol {
     
     /**
      Lists all caches.
-     - Returns: ListCachesResponse representing the result of the list caches operation.
+     - Returns: `ListCachesResponse` representing the result of the list caches operation.
+     
      Pattern matching can be used to operate on the appropriate subtype.
-    ```
-     switch response {
-     case let responseError as ListCachesError:
-        // handle error
-     case let responseSuccess as ListCachesSuccess:
-        // handle success
-        print(responseSuccess.caches)
-     }
-    ```
+     ```
+      switch response {
+      case .error(let err):
+         print("Error: \(err)")
+      case .success(let s):
+         print("Success: \(s.caches)")
```

Comment:
Oops, missed this one.
```suggestion
         print("Success: \(s.caches.map{ $0.name })")
```